src/EditorFeatures/Core/Extensibility/NavigationBar/WrappedNavigationBarItem.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Linq;
+using Microsoft.CodeAnalysis.NavigationBar;
+
+namespace Microsoft.CodeAnalysis.Editor
+{
+    /// <summary>
+    /// Implementation of the editor layer <see cref="NavigationBarItem"/> that wraps a feature layer <see cref="RoslynNavigationBarItem"/>
+    /// </summary>
+    internal class WrappedNavigationBarItem : NavigationBarItem
```

Comment:
there are now two concepts of a 'Navigation bar item', the 'feature layer' concept which is pure data, and the 'editor layer' which incorporates things like tracking spans.  We have to keep the latter around as it's what our partners produce.  So this type wraps the feature-layer version into the editor-layer-version.